src/foam/box/LocalBoxRegistry.js

```diff
@@ -63,14 +65,19 @@ foam.CLASS({
   methods: [
     {
       name: 'doLookup',
-      returns: 'foam.box.Box',
```

Comment:
Pretty sure we need this for stub generation, although maybe it doesn't matter cause its inherited from the interface?

Reply:
That's right. It figures it out from the interface.